appinventor/components/src/com/google/appinventor/components/runtime/ChartData2D.java

```diff
@@ -145,7 +166,7 @@ public Boolean call() {
   @SuppressWarnings("checkstyle:ParameterName")
   @SimpleFunction()
   public void ImportFromDataFile(final DataFile dataFile, String xValueColumn,
-      String yValueColumn) {
+                                 String yValueColumn) {
```

Comment:
Correct the indentation back to what it was. Continuations of parameter lines should be +4 from the start of the previous line.